### PR TITLE
Feature kafka producer

### PR DESCRIPTION
**Pull request by SrishtiR**
Added the producer to POST endpoint which publishes analyzed_tweets schema containing sentiment score to the topic 'trendSentiment'

### CODE REVIEW

File: application.py [hunk 1]
```diff
@@ -30,6 +32,12 @@
 
 aggregator = Aggregator(path)
 
+#connect to kafka producer
+producer = KafkaProducer(bootstrap_servers=['35.222.250.101:9092'],
```
Comment (AbigailFernandes) [inline application.py]: Shouldn't the IP be a part of the config

File: application.py [hunk 2]
```diff
@@ -120,6 +128,12 @@ def post_trend_sentiment():
             error = e
             log.error('[POST]/trendsentiment: Error in aggregaing the the tweets countrywise: ' + str(e))
```
Comment (AbigailFernandes) [inline application.py]: Aggregating spelling is incorrect

**Review by AbigailFernandes — CHANGES_REQUESTED** [review]

File: application.py (hunk 1 above)
Comment (SrishtiR) [reply application.py]: encrypted and added to config

File: application.py (hunk 2 above)
Comment (SrishtiR) [reply application.py]: corrected

**Review by juanhotencoding — APPROVED** [review]

**Review by shreshtha-pankaj — APPROVED** [review]

**Pull request merged**